frameworks/kafka/src/main/java/com/mesosphere/sdk/kafka/scheduler/KafkaService.java

```diff
@@ -29,13 +30,16 @@ public KafkaService(File pathToYamlSpecification) throws Exception {
                 DefaultScheduler.newBuilder(YAMLServiceSpecFactory.generateServiceSpec(rawServiceSpec));
         schedulerBuilder.setPlansFrom(rawServiceSpec);
 
-        /* Upgrade */
-        new KafkaConfigUpgrade(schedulerBuilder.getServiceSpec());
-        CuratorStateStoreFilter stateStore = new CuratorStateStoreFilter(schedulerBuilder.getServiceSpec().getName(),
+        /* Post Upgrade for version 1.1.21 */
+        CuratorStateStore stateStore = new CuratorStateStore(schedulerBuilder.getServiceSpec().getName(),
                 DcosConstants.MESOS_MASTER_ZK_CONNECTION_STRING);
-        stateStore.setIgnoreFilter(RegexMatcher.create("broker-[0-9]*"));
-        schedulerBuilder.setStateStore(stateStore);
-        /* Upgrade */
+        StringMatcher oldTaskNameFilter = RegexMatcher.create("broker-[0-9]*");
```

Comment:
Not sure I understand this: which versions would this upgrade from? 1.1.20 -> 1.1.21? Version 1.1.20 does not have 'broker-[0-9]' tasks, does it? Wouldn't it already have the new task names?